include/llvm/CodeGen/MachineMemOperand.h

```diff
@@ -76,7 +76,7 @@ struct MachinePointerInfo {
   /// Return a MachinePointerInfo record that refers to a GOT entry.
   static MachinePointerInfo getGOT(MachineFunction &MF);
 
```

Comment:
/// Return a MachinePointerInfo record that refers to a Page Offset Table (POT) entry.